src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -163,9 +173,15 @@ export class AppComponent implements OnInit, AfterViewChecked {
             this.navigateToGrid(this.activeGrid - 1);
         },
         'event.copySelection': () => {
-            let activeGrid = this.activeGrid;
-            let selection = this.slickgrids.toArray()[activeGrid].getSelectedRanges();
-            this.dataService.copyResults(selection, this.renderedDataSets[activeGrid].batchId, this.renderedDataSets[activeGrid].resultId);
+            let range: IRange = this.getSelectedRangeUnderMessages();
+            let messageText = range ? range.text() : '';
+            if (messageText.length > 0) {
+                this.executeCopy(messageText);
+            } else {
```

Comment:
Yeah that's how it works with this code, and I'd consider it a limitation / non ideal behavior. We could add a click event and explicitly deselect all active cells in the grid if this happens, but it's kinda hacky and not great. I've gone for "if message text is selected, use it, otherwise always copy cells". Not exactly perfect though. Let me know if you want a change and I can work to get click-based deselect working